vars/goMake.groovy

```diff
@@ -10,6 +10,8 @@ def call(body) {
     def ghOrg =  config.githubOrganisation
     def dockerOrg = config.dockerOrganisation
     def prj = config.project
+    def makeCommand = config.makeCommand ?: 'make'
+    def installPreRequisites = config.installPreRequisites
 
```

Comment:
ok, I was not sure if we should update the builder image as later sometime there can be many more pre-requisites like this our builder image will have many things installed. 
But yes surely faster builds and one time installation. I will update the builder image.